README.md

```diff
@@ -36,8 +36,9 @@ public object storage bucket and the name of the file containing the serialized
     ARG model_bucket=
     ARG model_file=
 
-Then, add the MD5 hashes of the files that will be downloaded to `md5sums.txt`. Note: the hashes should be of the files
-after any extraction (eg after un-taring or un-ziping).
+Then, calculate and add the MD5 hashes of the files that will be downloaded to `md5sums.txt`. Note: the hashes should be
+of the files after any extraction (eg after un-taring or un-ziping). To calculate the MD5 sum of a file run
```

Comment:
This is showing up weird on the README. I think formatting it like the following would look better

To calculate... run:
`$ <command here>`